src/style.css

```diff
@@ -1 +1,82 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+body {
+  box-sizing: border-box;
```

Comment:
It is a common practice to apply `box-sizing: border-box` for all elements via `*` selector